components/Form.jsx

```diff
@@ -81,13 +83,14 @@ const Form = ({formData, forNewMovie = true}) => {
           ]);
         }
       } else {
-        router.push("/");
+        return data.movie;
+        // router.push("/");
       }
     } catch (error) {
       console.log(error);
     }
   };
```

Comment:
You can extract postData into a js module in "lib/movieApi.js" file and call it as saveMovie()

Reply:
done